.github/workflows/publish-to-pypi.yml

```diff
@@ -1,19 +1,31 @@
 name: Publish Python 🐍 distributions 📦 to PyPI
 on: push
```

Comment:
Another solution would be to split this into 2 workflows.

One for prod, that would run only when a release is published:

```yaml
# publsh-to-pypi.yml
on:
  release:
    types: [published]

jobs:
...
```

And one for test, that would run every time there is a push to master:

```
# publsh-to-test-pypi.yml
on:
    push:
        branches:
          - master
...
```

this way we also don't need to skip anything with `if` conditions. I believe this might be a more elegant solution. What do you think?

Reply:
Nice!! Like it 😎